dedicated-examples/mock-camera-microphone-access/README.md

```diff
@@ -1 +1,29 @@
-# mock camera/microphone access
+# Mock Camera/Microphone Access
+
+This example shows how to mock data for microphone and camera. This example runs in **Chrome**.
+
+**Test Code**: [mock-camera-microphone-access.js](mock-camera-microphone-access.js)
+
+The test page requests a permission to record audio and video. Resulting streams are directed to corresponding `<audio>` and `<video>` elements on the page. When the `<button>` element registers a click, a snapshot of the video stream draws on the `<img>` element if video stream is present.
+
+The first test function uses [Selector.addCustomDOMProperties](https://devexpress.github.io/testcafe/documentation/reference/test-api/selector/addcustomdomproperties.html) method to read `<img>`'s `.src` property and check whether it holds an image.
+
+The second test function uses [Selector.addCustomDOMProperties](https://devexpress.github.io/testcafe/documentation/reference/test-api/selector/addcustomdomproperties.html) method to read `<audio>`'s `.srcObject` property. If audio stream is present, it returns `true`. This value is tested with [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) method.
```

Comment:
same here

Reply:
If audio stream is present -> If **an** audio stream is present

Reply:
"This value is tested with [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) method"? What do you mean 'is tested with'? How can something be tested with something else?